frame/staking/src/pallet/mod.rs

```diff
@@ -1015,7 +1030,7 @@ pub mod pallet {
 				}
 			}
 
-			Self::do_remove_nominator(stash);
+			let _ = Self::try_remove_nominator(stash);
```

Comment:
Why do we ignore return value?

Reply:
because we only want to make sure that they are no longer a nominator, not if they _were_ a nominator or not in the past. 

For example, this function can be called when an account is transitioning from: 

chilled -> validator
nominator -> validator

both if which are valid.